tests/unit/waiter-manager-noop-test.ts

```diff
@@ -24,14 +25,13 @@ if (!DEBUG) {
       assert.deepEqual(waiters, ['first']);
     });
 
-    test('a NoopTestWaiter always returns true from waitUntil', function(assert) {
+    test('a NoopTestWaiter always returns true from waitUntil', function(assert: Assert) {
       let waiter = buildWaiter('first');
-      let waiterItem = {};
 
       assert.ok(waiter.waitUntil(), 'waitUntil returns true');
-      waiter.beginAsync(waiterItem);
+      let token = waiter.beginAsync();
```

Comment:
Instead of changing the meaning of this test, we should add another test (IOW, we should test both custom token and generated token versions).

Reply:
👍